routers/api/v1/repo/issue.go

```diff
@@ -265,6 +272,16 @@ func EditIssue(ctx *context.APIContext, form api.EditIssueOption) {
 		issue.Content = *form.Body
 	}
 
+	var deadlineUnix util.TimeStamp
```

Comment:
nit: Use `var deadlineUnix util.TimeStamp` or `var deadline util.TimeStamp`, not both as it has same meaning (same in _repo/issue.go_ and _repo/pull.go_).

Reply:
Sure, I fixed that as well.